bugbot/crash/analyzer.py

```diff
@@ -717,9 +718,13 @@ def handler(search_resp: dict, data: list):
                         # Potential bad hardware crash, skip it.
                         continue
 
-                # TODO: Add a filter using the `possible_bit_flips_max_confidence`
-                # field to exclude bad hardware crashes. The filed is not available yet.
-                # See: https://bugzilla.mozilla.org/show_bug.cgi?id=1816669#c3
+                bit_flips_percentage = (
```

Comment:
Does this work as expected? In the UI:
* crash reports can have no bit flip probability mentioned: is it 0 or missing? [example](https://crash-stats.mozilla.org/signature/?product=Firefox&version=119.0&signature=js%3A%3Agc%3A%3Adetail%3A%3ACellHasStoreBuffer&date=%3E%3D2023-10-18T14%3A01%3A00.000Z&date=%3C2023-10-25T14%3A01%3A00.000Z&_columns=date&_columns=product&_columns=version&_columns=build_id&_columns=platform&_columns=reason&_columns=address&_columns=install_time&_columns=startup_crash&_columns=possible_bit_flips_max_confidence&_sort=-date&page=1#reports)
* In case it's missing, the scenario with every report having no value would need to be handled.

Reply:
> crash reports can have no bit flip probability mentioned: is it 0 or missing?

In theory, we should be relying on the confidence value. However, as discussed in [bug 1826345](https://bugzilla.mozilla.org/show_bug.cgi?id=1826345#c29), the confidence values are still not stable. Currently, as a workaround, we depend on the percentage of reports that have the confidence set (regardless of the value itself).

> in case it's missing, the scenario with every report having no value would need to be handled.

The idea now is that if 20% of the reports or more have a value in the `bit_flips_max_confidence` field, we ignore the signature.

Reply:
Just to be sure: `facets["cardinality_possible_bit_flips_max_confidence"]["value"]` will have a numeric value (`0`) is none of the crash reports has a bit flips confidence.

Reply:
Yes, it will be `0`. But there is another issue here. Using `cardinality` will count the distinct values of a field. All the crashes have the same value, it will be `1`. 

Fixed in 68a41d7b112d648a6f9c92f29f48f66a73b99c93.